tests/certificate/certificate.permission.test.js

```diff
@@ -33,13 +33,13 @@ describe('Run ApolloClientServer with role=admin in context', () => {
     adminContextServer = await setupApp();
 
     const certificateNullOwner = await generateCertificate(
-      { value: 1000 },
+      { value: 1000, count: 1 },
```

Comment:
How count would work if there are different values?

Reply:
We can only select one type of certificate

Reply:
on admin part we can use three of them at once.

Reply:
Ok, I will fix it out